payments/payments.Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:16
+WORKDIR /app
+COPY . .
+RUN npm install
+CMD ["node", "./dist/index.js"]
```

Comment:
breaking: `./dis/index.js` wont exist if `npm run build` is not first ran

`CMD` should be ["npm", "run", "start"] instead since `start` performs `npm run build && node ./dist/index.js`

- [ ] update `CMD`